packages/api/src/utility/connectUtility.js

```diff
@@ -62,14 +62,17 @@ async function connectUtility(driver, storedConnection, connectionMode, addition
 
     if (connection.sslCaFile) {
       connection.ssl.ca = await fs.readFile(connection.sslCaFile);
```

Comment:
Maybe assignement of connection.ssl.ca, connection.ssl.cert, connection.ssl.key can be removed?

It seems that now only file paths are used

Reply:
it's seems general connection options even for MySQL , PostgreSQL ? are they using blob/string of pem file instead of path. how about to keep blob/string of content also add extra path ?

Reply:
Of course you are right, I didn't catch that the first change is not in plugin